src/test/java/com/jcabi/xml/XMLDocumentTest.java

```diff
@@ -349,9 +349,10 @@ public void buildsDomNode() throws Exception {
      */
     @Test
     public void comparesToAnotherDocument() throws Exception {
+        final String xml = "<hi><dude>  </dude></hi>";
         MatcherAssert.assertThat(
-            new XMLDocument("<hi>\n<dude>  </dude></hi>"),
-            Matchers.equalTo(new XMLDocument("<hi><dude>  </dude></hi>"))
+            new XMLDocument(xml),
```

Comment:
this looks strange, we are comparing two instances of `XMLDocument` that are created using the same parameter, they will be always equal.
Can't we just leave the previus parameters here? (with new lines)